env/prepare.go

```diff
@@ -0,0 +1,65 @@
+package env
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+// PrepareWindowsOSEnv merges map of Windows registry environment variables with in-memory environment vars
+// Injects os.Getenv an os.Setenv
+func PrepareWindowsOSEnv(regEnvMap map[string]string, getenv func(string) string, setenv func(string, string) error) error {
+	for regKey, regValue := range regEnvMap {
+		envValue := getenv(regKey)
```

Comment:
`os.Getenv` on windows is case-insensitive so it will return the same for `PATH` or `Path`